### PR TITLE
implement multiple ldap servers

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
Behold, the copiest pastiest PR.

related to #307

### CODE REVIEW

File: awx/sso/conf.py [hunk 1]
```diff
@@ -129,271 +129,281 @@ def __call__(self):
...
+            ])),
+        ]),
+        feature_required='ldap',
+    )
+
+
+_register_ldap()
+_register_ldap('1')
+_register_ldap('2')
+_register_ldap('3')
```
Comment (cchurch) [inline awx/sso/conf.py:406]: What about 4 and 5?

File: awx/main/tests/functional/test_ldap.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
+
+import ldap
+import ldif
+import pytest
+import os
+from mockldap import MockLdap
```
Comment (cchurch) [inline awx/main/tests/functional/test_ldap.py:6]: Where is mockldap in requirements?

**Review by cchurch — CHANGES_REQUESTED** [review]

File: awx/sso/conf.py (hunk 1 above)
Comment (chrismeyersfsu) [reply awx/sso/conf.py:406]: Whoops. Fixed.

File: awx/main/tests/functional/test_ldap.py (hunk 2 above)
Comment (chrismeyersfsu) [reply awx/main/tests/functional/test_ldap.py:6]: Yep, missed that. Added.

**Review by cchurch — APPROVED** [review]

**Pull request merged**